src/main/java/seedu/address/logic/commands/DeleteAppointmentCommand.java

```diff
@@ -59,10 +61,21 @@ public CommandResult execute(Model model) throws CommandException {
             throw new CommandException(Messages.MESSAGE_INVALID_PERSON_INPUT);
         }
 
-        Person personWithoutAppointment = new Person(personToDeleteAppointment.getName(),
-                personToDeleteAppointment.getPhone(), personToDeleteAppointment.getEmail(),
-                personToDeleteAppointment.getTags(), new Appointment(new Date(""), new From(""), new To("")),
-                personToDeleteAppointment.getProperty());
+        Person personWithoutAppointment;
```

Comment:
Similar to editCommand. Perhaps extracting out the common inputs would clean up the code.